### DISCUSSION TITLE
Welcome to Altzone-WebPages Discussions!

### DISCUSSION THREAD

**Discussion by leolabdev** [Alt-Org/Altzone-WebPages#50] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Skoivumaki** [comment]
## Interesting Findings About How Our Fonts Are Set

I was looking for a way to add a new font used in Figma to our site and had a hard time getting it to work. After some troubleshooting, I found that our fonts are initially loaded in the topmost `layout.tsx` file:

```tsx
const sedgwickFont = Sedgwick_Ave_Display({
    subsets: ['latin'],
    weight: '400',
    variable: '--font-family-title',
    fallback: ['system-ui', 'arial'],
});

const urbanist = Urbanist({
    variable: '--font-family-main',
    subsets: ['latin'],
    weight: '700',
});

const rubik = Rubik({
    subsets: ['latin'],
    weight: '400',
    variable: '--font-family-texts',
});
```

These fonts are then applied to the root `<html>` tag, making them accessible across all pages. Logically, this should ensure that the loaded fonts are applied globally, and it does. However, since the app uses `global.scss` to modify font styles via CSS variables, I found an issue in the following lines:

```scss
// --font-family-texts: "Rubik";
// --font-family-main: 'Urbanist', sans-serif;
// --font-family-title: 'Sedgwick Ave Display', sans-serif !important;
```

If these are left uncommented, they seem to override the font variables set in `layout.tsx`, causing the initial font assignment to be lost. As a result, the styles default to sans-serif or whatever fallback font is specified.
The result being, the fonts wont actually be applied at all and only fallback will be used.

So the question is: have these issues been present for the past two years (time since the problematic code was added to `layout.tsx`), or did the fonts break at a later point? Personally, I haven't noticed any differences in how the fonts look since joining the project.  

Does anyone else remember or have any comments on this? If not, I suggest we address this issue and get our fonts working properly again.

### Example: Picture showing the diffrence in the Landing Page Between Old code and Fixed code. Both running in Production (`build`) Version

![image](https://github.com/user-attachments/assets/8c068308-fbc2-4735-8686-738edc9fb612)
